### DISCUSSION TITLE
Ability to initialize plugins without a handler.lua and schema.lua

### DISCUSSION THREAD

**Discussion by jvandervelden** [Kong/kong#7267] [Ideas and feature requests]
We are trying to add a plugin with only an admin api extension. It would be nice to not have to specify a handler.lua and a schema.lua because it's not that type of plugin. And just having the plugin in the lua cache doesn't get picked up automatically.

### Steps To Reproduce

1. Create a plugin with only api.lua
2. Add plugin to plugins field in the configuration
3. Start Kong

### Additional Details & Logs

- Kong version 1.0.0
- Operating system: Docker on Windows

**Comment by bungle** [comment]
@jvandervelden, thanks! surely something to consider!

**Comment by thibaultcha** [comment]
Indeed, this idea has been around for as far as a few years ago. We might give it full consideration someday; as always, we need to balance the project's priorities. We'd also be happy to draft some specifications with community members and expect as PR!
